cmd/cape/cmd/run.go

```diff
@@ -137,39 +146,30 @@ func run(cmd *cobra.Command, args []string) error {
 }
 
 // This function is exported for tuner to use.
-func Run(url string, functionID string, file string, insecure bool) ([]byte, error) {
+func Run(url string, token string, functionID string, file string, insecure bool) ([]byte, error) {
 	input, err := os.ReadFile(file)
 	if err != nil {
 		return nil, fmt.Errorf("unable to read data file: %w", err)
 	}
 
+	a := entities.FunctionAuth{
+		Token: token,
+		Type:  entities.AuthenticationTypeAuth0,
+	}
```

Comment:
Is single character identifiers idiomatic in Go? I tend to prefer having more descriptive identifier names if possible, such as `auth`.

Reply:
As long as they aren't in scope for a many lines, yes